index.html

```diff
@@ -1,120 +1,649 @@
-                {"name": "Money C", "data": [8000, 3000, 10000, 6000]}
-            ]
-        }, {
-            plugins: [
-                Chartist.plugins.legend()
-            ]
-        });
-    </script>
+   <div style="max-width: 700px; margin: 0 auto 40px auto;">
+      <h1>chartist-plugin-legend examples</h1>
+      <h2>basic examples</h2>
+      <h3>Line Chart:</h3>
+      <div class="ct-chart ct-chart-line"></div>
+      <button class="button-showcode">Show Code</button>
+      <pre class="codeblock-hidden"><code class="javascript">new Chartist.Line('.ct-chart-line', {
+   labels: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'],
+   series: [
+       { "name": "Money A", "data": [12, 9, 7, 8, 5] },
+       { "name": "Money B", "data": [2, 1, 3.5, 7, 3] },
+       { "name": "Money C", "data": [1, 3, 4, 5, 6] }
+   ]
+}, {
+   fullWidth: true,
+   chartPadding: {
+      right: 40
+   },
+   plugins: [
+       Chartist.plugins.legend()
+   ]
+});</code></pre>
+      <h3>Pie Chart:</h3>
+      <div class="ct-chart ct-chart-pie"></div>
+      <button class="button-showcode">Show Code</button>
+      <pre class="codeblock-hidden"><code class="javascript">new Chartist.Pie('.ct-chart-pie', {
+    labels: ['Piece A', 'Piece B', 'Piece C', 'Piece D'],
+    series: [20, 10, 30, 40]
+}, {
+    showLabel: false,
+    plugins: [
+        Chartist.plugins.legend()
+    ]
+});</code></pre>
+      <h3>Bar Chart:</h3>
+      <div class="ct-chart ct-chart-bar"></div>
+      <button class="button-showcode">Show Code</button>
+      <pre class="codeblock-hidden"><code class="javascript">new Chartist.Bar('.ct-chart-bar', {
+   labels: ['First quarter of the year', 'Second quarter of the year', 'Third quarter of the year', 'Fourth quarter of the year'],
+   series: [
+       { "name": "Money A", "data": [60000, 40000, 80000, 70000] },
+       { "name": "Money B", "data": [40000, 30000, 70000, 65000] },
+       { "name": "Money C", "data": [8000, 3000, 10000, 6000] }
+   ]
+}, {
+   plugins: [
+       Chartist.plugins.legend()
+   ]
+});</code></pre>
+      <h2>examples with options</h2>
+      <h3>Chart with <i>className</i>:</h3>
+      <span>Adds a class to the <code>ul</code> element.</span>
+      <div class="ct-chart-line-classname"></div>
+      <button class="button-showcode">Show Code</button>
+<pre class="codeblock-hidden"><code class="javascript">new Chartist.Line('.ct-chart-line-classname', {
+   labels: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'],
+   series: [
+       { "name": "Money A", "data": [12, 9, 7, 8, 5] },
+       { "name": "Money B", "data": [2, 1, 3.5, 7, 3] },
+       { "name": "Money C", "data": [1, 3, 4, 5, 6] }
+   ]
+}, {
+   fullWidth: true,
+   chartPadding: {
+      right: 40
+   },
+   plugins: [
+       Chartist.plugins.legend({
+          className: 'crazyPink'
+       })
+   ]
+});</code></pre>
+      <h3>Chart with <i>clickable</i>:</h3>
```

Comment:
Yeah, i actually changed the value to `false`, so actual changes for the option can be seen on the page; it would be quite irritating not to change anything, this way the viewer won't see any changes the option makes, but the description is indeed misleading. I'd change the description to:

Sets the legends clickable state; setting this value to false disables toggling graphs on legend click.

Please let me know if you can agree with that making sense.